i18n/en.json

```diff
@@ -2453,6 +2453,8 @@
   "apps.error.parser.multiple_equal": "Multiple `=` signs are not allowed.",
   "apps.error.parser.no_argument_pos_x": "Unable to identify argument.",
   "apps.error.parser.no_bindings": "No command bindings.",
+  "apps.error.parser.no_call": "No call found.",
+  "apps.error.parser.no_form": "No form found.",
```

Comment:
There's a ticket for improving the user-facing error messages in the command parser https://mattermost.atlassian.net/browse/MM-35315. The ticket is vague, but 1/5 part of what needs to be done is removing the developer-level translations, and log the debug messages to the console instead. I'm not sure if we have tooling in place specifically for logging to the console, because I assume `console.log` may not be allowed by the linter.

Reply:
If we want to log things into the console, we can (we just add a linter ignore). But I don't know what are the general thoughts around that. What other error message looks better in this place than these? I mean... they are not really useful for the end-user, but at least they are useful when they ask the app developer, without more turnarounds like "reproduce it again and look in the developer console" -> "the developer what?", etc...